### PR TITLE
[BACKLOG-10490] - Create and Dev Validate the cdh58 shim against Pentaho 7.0 (master) - add cdh58

### PR DESCRIPTION
**Pull request by alexzhk**
Added cdh58 to the following pom files:

 - assemblies/legacy-plugin/pom.xml;
 - pom.xml.

### CODE REVIEW

**Comment by alexzhk** [comment]
Could you @dkincade @hudak please review my changes?

File: assemblies/legacy-plugin/pom.xml [hunk 1]
```diff
@@ -164,6 +164,13 @@
     </dependency>
     <dependency>
       <groupId>pentaho</groupId>
+      <artifactId>pentaho-hadoop-shims-cdh58-package</artifactId>
```
Comment (dkincade) [inline assemblies/legacy-plugin/pom.xml]: Please remove CDH 5.5 (above)

File: pom.xml [hunk 2]
```diff
@@ -58,6 +58,7 @@
     <dependency.jets3t.revision>0.9.3</dependency.jets3t.revision>
 	<dependency.hadoop-shims-cdh55.revision>7.0-SNAPSHOT</dependency.hadoop-shims-cdh55.revision>
 	<dependency.hadoop-shims-cdh57.revision>7.0-SNAPSHOT</dependency.hadoop-shims-cdh57.revision>
+    <dependency.hadoop-shims-cdh58.revision>7.0-SNAPSHOT</dependency.hadoop-shims-cdh58.revision>
```
Comment (dkincade) [inline pom.xml]: Please remove CDH 5.5 (above)

File: assemblies/legacy-plugin/pom.xml (hunk 1 above)
Comment (alexzhk) [inline assemblies/legacy-plugin/pom.xml]: Removed cdh 5.5 and updated assemblies/legacy-plugin/pom.xml

File: pom.xml (hunk 2 above)
Comment (alexzhk) [inline pom.xml]: Removed cdh 5.5 and updated pom.xml

**Pull request merged**